<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rentReturnWindow</class>
 <widget class="QMainWindow" name="rentReturnWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
background-color: rgb(47, 165, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="3" column="0">
     <widget class="QTextEdit" name="statistics">
      <property name="styleSheet">
       <string notr="true">*{
background-color: rgb(255, 255, 255);
	border-color: rgb(252, 64, 51);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="lineWrapMode">
        <enum>QTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="4">
       <widget class="QLabel" name="rentedLb">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
color: rgb(255, 79, 35);
}</string>
        </property>
        <property name="text">
         <string>RENTED BOAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="availableLb">
        <property name="font">
         <font>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
color: rgb(255, 79, 35);
}</string>
        </property>
        <property name="text">
         <string>AVAILABLE BOAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="selBret">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 13pt &quot;Helvetica&quot;;</string>
          </property>
          <property name="text">
           <string>Select boat to return</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="rentedCB">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
	font: 13pt &quot;Helvetica&quot;;
	background-color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="4">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="retBbtn">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
	font: 13pt &quot;Helvetica&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));

}</string>
          </property>
          <property name="text">
           <string>RETURN  BOAT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cal">
          <property name="styleSheet">
           <string notr="true">*{
	font: 13pt &quot;Helvetica&quot;;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));

}</string>
          </property>
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="time">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <item>
           <property name="text">
            <string>AM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PM</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="selBren">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Select boat to rent:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="indent">
           <number>160</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="availableCB">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">*{
	font: 13pt &quot;Helvetica&quot;;
	background-color: rgb(255, 255, 255);
}</string>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAlphabetically</enum>
          </property>
          <item>
           <property name="text">
            <string>Boat #1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #10</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #11</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #12</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #13</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #14</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Boat #15</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="addBbtn">
          <property name="styleSheet">
           <string notr="true">*{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
	font: 13pt &quot;Helvetica&quot;;
}</string>
          </property>
          <property name="text">
           <string>Add Boat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delBbtn">
          <property name="styleSheet">
           <string notr="true">*{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
	font: 13pt &quot;Helvetica&quot;;
}</string>
          </property>
          <property name="text">
           <string>Delete Boat</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="takeBbtn">
          <property name="font">
           <font>
            <family>Helvetica</family>
            <pointsize>13</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">*{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));
	font: 13pt &quot;Helvetica&quot;;
}</string>
          </property>
          <property name="text">
           <string>TAKE BOAT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="rrWin">
        <property name="font">
         <font>
          <family>Spooked</family>
          <pointsize>33</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
color: rgb(0, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>RENT AND RETURN WINDOW</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>75</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="ttAvgLb">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>30</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{
	font: 30pt &quot;Helvetica&quot;;
color: rgb(255, 79, 35);
}</string>
        </property>
        <property name="text">
         <string>Average Time</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ttAvgDisp">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>100</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">*{color:rgb(85, 0, 0); background-color: rgb(255, 255, 255);
font: 14pt &quot;Helvetica&quot;;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="timeTracking">
      <item row="1" column="0">
       <widget class="QLabel" name="ttBoat">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>BOAT</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>138</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="ttDispBoat">
        <property name="styleSheet">
         <string notr="true">*{
background-color: rgb(255, 255, 255);
	font: 13pt &quot;Helvetica&quot;;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="ttStart">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>TIME(Rented/Added/Deleted/Lost)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="ttDispStart">
        <property name="styleSheet">
         <string notr="true">*{
background-color: rgb(255, 255, 255);
	font: 13pt &quot;Helvetica&quot;;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>50</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="ttTotal">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>TIME(Current/Returned)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>85</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="ttDispEnd">
        <property name="styleSheet">
         <string notr="true">*{
background-color: rgb(255, 255, 255);
	font: 13pt &quot;Helvetica&quot;;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
